### ISSUE TITLE
"Translate" could be changed to "upgrade"?

### ISSUE THREAD

**Issue by kerimoyle** [libcellml/website-src#67]
I find the translate block and its icon misleading!  Can we change it to "upgrade" or "update" or something a bit more specific to CellML and not related to human languages?

**Comment by hsorby** [comment]
I prefer translate as that is exactly what it is doing.  The icon is a a generic one from MDI so has some familiarity and CellML is a language.  For me all of that reinforces what the user can expect from that action item.

**Comment by kerimoyle** [comment]
Yeah, the MDI icon is familiar for translating human languages, not code!  That's exactly what the problem is for me.  It's not even changing from CellML to another language format, it's just changing from one version of CellML to another ... and isn't that what an upgrade is?

**Comment by agarny** [comment]
Personally, I would say "convert". :)

**Comment by hsorby** [comment]
Technically it is performing a transformation.
